### PR TITLE
Add kanban board cards and assign work

### PR DESCRIPTION
**Pull request by imranhossainemi**
**Project update**

- Add kanban board 
- Assign work 
- Initial react redux file structure  
- Issue link https://github.com/imranhossainemi/space_travelers_hub/issues/26
- kanban board link https://github.com/imranhossainemi/space_travelers_hub/projects/1

### CODE REVIEW

**Comment by rloterh** [comment]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Please ensure your project is submitted using **GitHub issues**, and your description should contain the following info:
  * link to their GitHub project
  * information about the number of people in their group

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by emmanuellekamwa** [comment]
**STATUS: INVALID ❌⛔🛑**
Hello @imranhossainemi and @byiringiroscar 👋
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Your pull request lacks the link to the GitHub project and the number of people participating in the project just like the previous reviewer mentioned.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by imranhossainemi** [comment]
> **STATUS: INVALID ❌⛔🛑** Hello @imranhossainemi and @byiringiroscar 👋 While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> ## Invalid Code Review Request
> 
> Your pull request lacks the link to the GitHub project and the number of people participating in the project just like the previous reviewer mentioned.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers, and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Can you show me with an example what I have to do?

File: package.json [hunk 1]
```diff
@@ -35,5 +35,22 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
```
Comment (HAMMAS-SALEEM) [inline package.json:38]: image

- [ ] As you can see in the image above that some requirements are missing 😨 First you should open a new issue in this repository and then include the following information in that issue 🤔 

* Github Project Link (Link to Kanban Board you have created)
* Information about the total number of members in this project

## Example ⬇️
image

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submission :red_circle:

invalid-submission

## Hey @imranhossainemi, @byiringiroscar :wave:

Good job on implementing some of the requirements but I cannot proceed review of your pull request because there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**